README.md

```diff
@@ -160,9 +160,9 @@ and a placeholder token (`^`) standing in for the previous operation’s value:
 Object.keys(envars)
   .map(envar => `${envar}=${envars[envar]}`)
   .join(' ')
-  |> `$ ${^}`
-  |> chalk.dim(^, 'node', args.join(' '))
-  |> console.log(^);
+    |> `$ ${^}`
```

Comment:
These lines are purposely indented the same amount as the methods. I expect this might become common indenting practice.